migration/idempotent/001-base.sql

```diff
@@ -2608,13 +2619,21 @@ BEGIN
     OR (column_name = 'series_id' AND data_type = 'bigint')
     OR data_type = 'double precision');
 
+    -- We only handle automatic refresh if refresh_interval is applied. Otherwise
+    -- we ask the user to care about refreshing this Cagg.
+    IF refresh_interval IS NULL THEN
```

Comment:
Yes we could, but we shouldn't. I think we should rather error out, saying if this is a rollup, then it has to have a refresh_interval. We shouldn't try to repair here, since a rollup without a refresh_interval means our internal code is messed up.